lisa/src/test/java/it/unive/lisa/test/imp/tutorial/ParityXSign.java

```diff
@@ -0,0 +1,20 @@
+package it.unive.lisa.test.imp.tutorial;
+
+import it.unive.lisa.analysis.ValueCartesianProduct;
+import it.unive.lisa.analysis.nonrelational.ValueEnvironment;
+import it.unive.lisa.symbolic.value.Identifier;
+
+public class ParityXSign extends ValueCartesianProduct<ValueEnvironment<Parity>, ValueEnvironment<Sign>> {
```

Comment:
Do we really need this class? Can't we just use `new ValueCartesianProduct(new Pairity(), new Sign())` instead of redefining a new instance of cartesian product each time?